### PR TITLE
Delete useless comment

### PR DESCRIPTION
**Pull request by BaiShaoqi**
The confusing comment is spotted by Asim R P, and he gaves another comment on it.

After quering the author about the confusing comment,  the confusing comment could be deleted.

### CODE REVIEW

File: src/backend/commands/extension.c [hunk 1]
```diff
@@ -1260,10 +1260,6 @@ CreateExtension(CreateExtensionStmt *stmt)
 							stmt->extname)));
 	}
 
-	/* We want to promote user to superuser in QE when creating extension */
-	/* GPDB_91_MERGE_FIXME: I don't see any such promotion here. But this is doing
```
Comment (ashwinstar) [inline src/backend/commands/extension.c]: What is that something else important? can we document that.

Comment (BaiShaoqi) [reply src/backend/commands/extension.c]: Sorry, ashwinstar,  I did not confirm "something else important"  with Asim R P.
He may answer it later

Comment (zhangh43) [reply src/backend/commands/extension.c]: QE is already run by superuser, right?

**Comment by BaiShaoqi** [comment]
Thanks for yours reviews

**Pull request merged**